baseapp_profiles/graphql/mutations.py

```
@@ -286,12 +413,244 @@ def mutate_and_get_payload(cls, root, info, **input):
+                        invited_at=timezone.now(),
+                        invitation_expires_at=(
+                            timezone.now() + timedelta(days=INVITATION_EXPIRATION_DAYS)
+                        ),
+                    )
+                    invitation.generate_invitation_token()
+                    invitation.save()
+                except IntegrityError:
+                    raise GraphQLError(
+                        str(_("An invitation has already been sent to this email")),
+                        extensions={"code": "duplicate_invitation"},
+                    )
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)